utils/tests/test_system_check.py

```diff
@@ -0,0 +1,129 @@
+from sheetstorm.system_check import create_error_02_invalid_domain_address
+from sheetstorm.system_check import create_error_04_site_id_value_must_be_integer
+from sheetstorm.system_check import create_error_05_email_backend_value_must_be_string
+from sheetstorm.system_check import create_error_06_email_backend_library_does_not_exist
+from sheetstorm.system_check import create_error_07_email_backend_file_does_not_have_attribute
+from sheetstorm.system_check import create_warning_01_duplicated_values_in_valid_email_domain_list
+from sheetstorm.system_check import create_warning_02_empty_valid_email_domain_list
+
+
+class SystemCheckTest(TestCase):
+    @override_settings(VALID_EMAIL_DOMAIN_LIST=["validdomain.it", "codepoets.it"])
+    def test_check_settings_valid_email_domain_list_with_correct_list(self):
+        errors = check_settings_valid_email_domain_list()
+
+        self.assertEqual(errors, [])
+
+    @override_settings(VALID_EMAIL_DOMAIN_LIST=["duplicate.com", "duplicate.com"])
+    def test_check_settings_valid_email_domain_list_warning_message_when_list_contains_duplicates(self):
+        errors = check_settings_valid_email_domain_list()
+
+        self.assertTrue(len(errors) > 0)
+        self.assertEqual(errors[0], create_warning_01_duplicated_values_in_valid_email_domain_list())
+
+    @override_settings(VALID_EMAIL_DOMAIN_LIST=666)
+    def test_check_settings_valid_email_domain_raise_error_when_it_is_not_instance_of_list(self):
+        errors = check_settings_valid_email_domain_list()
+
+        self.assertTrue(len(errors) > 0)
+        self.assertEqual(errors[0], create_error_01_valid_email_domain_list_is_not_a_list())
+
+    @override_settings(VALID_EMAIL_DOMAIN_LIST=[])
+    def test_check_settings_valid_email_domain_warning_message_when_list_is_empty(self):
+        errors = check_settings_valid_email_domain_list()
+
+        self.assertTrue(len(errors) > 0)
+        self.assertEqual(errors[0], create_warning_02_empty_valid_email_domain_list())
+
+    @override_settings(VALID_EMAIL_DOMAIN_LIST=["localhost"])
+    def test_check_settings_valid_email_domain_raise_error_when_domain_doesnt_have_dot(self):
+        errors = check_settings_valid_email_domain_list()
+        domain = settings.VALID_EMAIL_DOMAIN_LIST[0]
+
+        self.assertTrue(len(errors) > 0)
+        self.assertEqual(errors[0], create_error_02_invalid_domain_address(domain))
+
+    @override_settings(VALID_EMAIL_DOMAIN_LIST=["code/@,poets.it"])
+    def test_check_settings_valid_email_domain_raise_error_when_domain_contains_prohibited_signs(self):
+        errors = check_settings_valid_email_domain_list()
+        domain = settings.VALID_EMAIL_DOMAIN_LIST[0]
+
+        self.assertTrue(len(errors) > 0)
+        self.assertEqual(errors[0], create_error_02_invalid_domain_address(domain))
+
+    @override_settings()
+    def test_check_settings_valid_email_domain_raise_error_when_settings_doesnt_exist(self):
```

Comment:
this is wrong. Please write code that would check if `VALID_EMAIL_DOMAIN_LIST` exist in settings and returns custom `Error` if not